src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/Editor_OutOfProc.cs

```diff
@@ -169,5 +169,44 @@ public void WaitForActiveView(string viewName)
 
         public string[] GetErrorTags()
             => _editorInProc.GetErrorTags();
+
+        public void ExpandProjectNavBar()
```

Comment:
This is our opportunity to call them something better than "Project/Left/Right" here, since the "Left/Right" part is named that way for legacy reasons, and "Left" actually means "Middle". Project/Type/Member?